_docs/_latest/concepts/models.md

```diff
@@ -1,84 +1,77 @@
-relationships.  At any given time, multiple set of tables can exist,
-either created by the cron job, or by other users. The table sets are
-listed in the `models` table.  
+## How data models are stored
 
-### binding_members
+Data models are stored in its own set of tables, which are named with the
+`<model_handle>_<table_name>` and are tied to each other by specific relationships.  At any given
+time, multiple set of tables can exist, either created by the cron job, or by other users. The
+table sets are listed in the `models` table.  
 
-This table is a join table that connects `members` table with the `bindings`
-table, thus making it possible to know what resources each member can access.
+### The `binding_members` table
 
-### bindings
+This table is a join table that connects `members` table with the `bindings` table, thus making it
+possible to know what resources each member can access.
 
-This table contains information about what resource and what role are
-associated for a `binding_id`.
+### The `bindings` table
 
-Thus by combining this with `binding_members` table, we can see who has
-access to what resources, and with which roles.
+This table contains information about what resource and what role are associated for a
+`binding_id`. By combining this with `binding_members` table, we can see who has access to what
+resources, and with which roles.
 
-### group_in_group
+### The `group_in_group` table
 
 This table contains information about how groups are nested in other groups.
 Each row contains a group and its parent group.
 
 If a group is not nested, then it will not be in this table.
 
-### group_members
+### The `group_members` table
 
 This table contains information about groups, and the members in the group
 (both users and other groups).
 
-### members
+### The `members` table
 
 This table contains information about members, the types of resources they are,
 and their names.
 
-### permissions
+### The `permissions` table
 
 This table is a listing of all the permissions on GCP.
 
-### roles
+### The `roles` table
 
 This table is a listing of all the roles on GCP, title, stage, description,
-whether it's custom role or not.
+and whether it's custom role or not.
 
-### role_permissions
+### The `role_permissions` table
 
 This table contains information on the roles and the permissions of that role.
 
-By combining the `binding_members`, `bindings`, `roles`, and `role_permissions`
-tables, we can see who has what permissions on what resources.
+By combining the `binding_members`, `bindings`, `roles`, and `role_permissions` tables, we can see
+who has what permissions on what resources.
 
-### resources
-This table contains the details of each resource, such as `full_name`,
-its parents, and raw gcp data.  This is the table that the various scanners use
-to do all the scannings.
+### The `resources` table
+This table contains the details of each resource (e.g., `full_name`, its parents, and raw GCP data).
```

Comment:
Don't use Latin abbreviations: "This table contains the details of each resource, like the `full_name`, its parents, and raw GCP data."

Reply:
done